forms-html/RevanAndShadan/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        
+    </head>
+    <body>
+        <form >
+            <label for="email" style="font-size: 32px;">Enter your email:</label><br> 
+
+            <input type="email" id="email" name="email" value="osama@re-coded.com"><br>
+            <label for="username" style="font-size: 32px;">Username:</label>
+         <input type="text" id="username" name="username" value="osama"><br>
+            <label for="password" style="font-size: 32px;">Password</label><br>
+        <input type="password" id="password" name="password" value="123111"> <br>
+        <label for="number">Quantity (between 1 and 5)</label>
+        <input type="number" id="number" name="number" value="3" max="5" min="1">
+        <label for="range">Volume (between 0 and 50)</label>
+        <input type="range" id="range" name="range" min="0" max="50">
+        <h3>radio</h3>
+        <input type="radio" id="male" name="gender" value="Male" checked><label for="male">Male</label><br>
+        <input type="radio" id="female" name="gender" value="Female"><label for="female">Female</label><br>
+        <input type="radio" id="other" name="gender" value="other"><label for="other">Other</label><br>
+        
+        <label>checkbox</label> 
+        <input type="checkbox" name="check" value="bike" checked> <label for="bike" > I have a bike</label><br>
+        <input type="checkbox" name="check" value="car" > <label for="bike" > I have a car</label><br>
+        <input type="checkbox" name="check" value="boat" > <label for="bike" > I have a boat</label><br>
+      <label>button</label>
+       <input type="button" value="CLick Me !"> <br>
+       <label for="color">Select your favorate color</label>
+       <input type="color" id="color" name="color"><br>
+       <label for="date">Enter the date before 1980-1-1</label>
```

Comment:
the date before 1980-1-1 should validate the date and don't accept date before that